app/src/cli/mod.rs

```diff
@@ -101,6 +101,10 @@ pub struct HostCli {
     /// Defaults to 9000
     #[arg(long)]
     pub quic_interface_port: Option<u16>,
+
+    /// AmbientProxy address to use for NAT traversal
+    #[arg(long)]
+    pub proxy: Option<String>,
```

Comment:
Can we make it so it defaults to proxying? I.e. when you run `ambient serve` or `ambient run` it just prints something like "use `ambient join proxy.ambient.run:324` to join". We can add a `--no-proxy` flag to turn it off.

I know this might be a bit "wasteful" in the future if there's a lot of local development and we can see about revisiting it then, but I just want to make it as easy as possible for people to create multiplayer experiences now. (Though we can probably hole-punch in the future also, right?)

Reply:
fwiw Gradio uses `--share` for this; applications are local by default, but you're given a proxy URL when you specify `--share` (no arguments). 

That seems like a reasonable path to me - I think people would prefer development to be local by default (I've definitely had my fair share of people joining my development servers when I didn't expect it!)

Reply:
Yeah, fair enough, let's see how it goes 👍